stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/database_logic.py

```diff
@@ -39,7 +39,7 @@
     ":",
 }
 
-DEFAULT_INDICES = f"*,-*kibana*,-{COLLECTIONS_INDEX}"
```

Comment:
This won't work as intended and isn't backwards-compatible if the ITEMS_INDEX_PREFIX is set to `""`, as is the intention when cross-compatibility with stac-server is desired.

Reply:
Is there a solution? I should have waited to merge this.

Reply:
I re-added the exclusion of the internal `*kibana*` indices and the collection index with a wildcard to support numbered indices with an alias.